### PR TITLE
Fix yarn installb

### PR DESCRIPTION
**Pull request by dwu359**
# Fix yarn installb

**What user problem are we solving?**
yarn installb gave an error if the conda environment is already created. The pre-commit cmd would also fail if the dev wasn't logged in to GitGuardian.

**What solution does this PR provide?**
yarn installb updates the environment if it detects that the environment is already created, and prompts the dev to sign in to GitGuardian if it isn't.

**Testing Methodology**
How did you test your changes and verify that existing 
functionality is not broken
I tested it by deleting my conda env and reinstalling it with yarn installb

**Any other considerations**

### CODE REVIEW

**Review by karkir0003 — COMMENTED** [review]
what does the --prune tag do?

File: package.json [hunk 1]
```diff
@@ -6,7 +6,7 @@
   "scripts": {
     "build:prod": "cd frontend && yarn install && REACT_APP_MODE=prod yarn run build",
     "installf": "cd frontend && yarn install",
-    "installb": "conda --version && cd conda && conda env create -f environment.yml && pre-commit install",
+    "installb": "conda --version && cd conda && if conda env list | grep -q 'dlplayground'; then conda env update -f environment.yml --prune; else conda env create -f environment.yml; fi && ggshield auth login && pre-commit install",
```
Comment (karkir0003) [inline package.json:9]: what does --prune do?

Comment (karkir0003) [inline package.json:9]: did the latency of creating conda env get affected? were you able to verify that with newly created conda env, the backend works correctly and no weird errors

**Review by farisdurrani — APPROVED** [review]

**Comment by farisdurrani** [comment]
> what does the --prune tag do?

I believe that's for pruning (uninstalling) unused packages

**Comment by farisdurrani** [comment]
Approved, you may merge the branch anytime

**Review by karkir0003 — APPROVED** [review]
looks good to me

File: package.json (hunk 1 above)
Comment (dwu359) [reply package.json:9]: Yep, the backend works correctly without errors when I ran yarn installb for a new env

**Pull request merged**